core/metrics-core-service/src/main/java/org/hawkular/metrics/core/dropwizard/MetricNameService.java

```diff
@@ -33,28 +33,45 @@
  */
 public class MetricNameService implements MetricFilter {
 
-    public static final String TENANT_ID = "admin";
 
     private static final String separator = ":";
 
-    private String hostname;
+    private final String adminTenant;
+    private final String hostname;
 
     public MetricNameService() {
         try {
             hostname = InetAddress.getLocalHost().getHostName();
         } catch (UnknownHostException e) {
             throw new RuntimeException("Failed to get hostname", e);
         }
+
+        this.adminTenant = "admin";
```

Comment:
@stefannegrea you might want to rebase on master because the version of this class is out of date a bit. The constructor is used for testing, but admin tenant is still hard coded, 

```java
public static final String TENANT_ID = "admin";
```

That will have to change.